lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -1922,6 +1930,12 @@ bool TFFunctionPartition::markFunction() {
     }
   }
 
+  // If the function is convention(tensorflow), arguments should be marked Move.
```

Comment:
Please s/convention/@convention/

Reply:
What does this code chunk achieve? (i.e., if we remove it, what test would fail?)

my understanding is that func arguments must become graph function arguments for an accelerator-only function -- do we have to mark it anything in order to achieve that?

Marking::Move is applied to statements that are *within* the tensor program region, but here the program region has not started.

Reply:
This makes the partitioner not arguments as copy-in, because `@convention(tensorflow)` already expects arguments to be on the accelerator. If we remove it, there will be implicit copy warnings.

> Marking::Move is applied to statements that are within the tensor program region, but here the program region has not started.

Oops, my comment wasn't clear. It's actually marking the *users* of each argument to be Move.

Reply:
Marking function argument users as Move (or Argument) turns out to be problematic. It's hitting a trap in the PartitionCloner. I changed the code to disable implicit-copy-to-accelator diagnostics for function arguments when the function is accelerator-only. This can be generalized later.